### ISSUE TITLE
Questions About Training Settings

### ISSUE THREAD

**Issue by dingyue772** [StarDewXXX/O1-Pruner#4]
Thank you for open-sourcing such an excellent project! I have some questions regarding your training settings.

- How did you choose the `finetuning_type` in LlamaFactory for your experiments?
- Have you tried using the O1-Pruner method on smaller models, such as DS_Distilled_Qwen_1.5B?
I followed the same training settings as QWQ32B and observed a decreasing loss curve. However, maybe since I used a very small dataset (50 problems, 100 samples), the trained model exhibited increased response length and accuracy.

Additionally, when fine-tuning the Marco-o1 model using LoRA with 500 problems and 1000 samples, I observed a highly fluctuating loss curve. Could this be due to an inappropriate number of training epochs or the `finetuning_type `setting or maybe still due to the `dataset size`? 

Would you share more training details using Llama-Factory? 

I would appreciate any insights you could share. Thanks in advance!
